modules/client/src/Connext.ts

```diff
@@ -98,17 +99,23 @@ export interface IHubAPIClient {
   updateThread(update: ThreadStateUpdate): Promise<ThreadStateUpdate>
   getLatestChannelStateAndUpdate(): Promise<{state: ChannelState, update: UpdateRequest} | null>
   getLatestStateNoPendingOps(): Promise<ChannelState | null>
+  config(): Promise<HubConfig>
 }
 
 class HubAPIClient implements IHubAPIClient {
   private user: Address
   private networking: Networking
-  private tokenName?: string
+  // private tokenName?: string
 
   constructor(user: Address, networking: Networking, tokenName?: string) {
     this.user = user
     this.networking = networking
-    this.tokenName = tokenName
+    // this.tokenName = tokenName
```

Comment:
should delete the reference here if were not using it, minor tweak

Reply:
Yea, I've seen this not being used for a while now.